config/locales/en.yml

```diff
@@ -89,6 +89,15 @@ en:
           msg: "There's no meeting currently running in this room"
           title: "Meeting not running"
     bigbluebutton: BigBlueButton
+    meetings:
+      delete:
+        success: "The meeting was successfully deleted."
+      notice:
+        destroy:
+          success_with_bbb_error: "The meeting was successfully destroyed but it wasn't deleted from the webconference server (\"%{error}\")"
+          error_destroy: "This meeting could not be deleted"
+          running:
+            not_ended: "This meeting can not be deleted because it is still running"
```

Comment:
new message: "This register can not be deleted because it is still running"

Reply:
done